mmcv/runner/hooks/evaluation.py

```diff
@@ -95,6 +109,24 @@ def __init__(self,
         self.eval_kwargs = eval_kwargs
         self.initial_flag = True
 
+        if test_fn is None:
+            from mmcv.engine import single_gpu_test
+            self.test_fn = single_gpu_test
+        else:
+            self.test_fn = test_fn
+
+        if greater_keys is None:
+            self.greater_keys = self._default_greater_keys
+        else:
+            assert is_list_of(greater_keys, str)
```

Comment:
Do we need to also support greater_keys to be a string like 'mIoU'?

Reply:
We may check if `isinstance(greater_keys, (list, tuple))`, otherwise, make it into tuple.